interpreter/cling/lib/Interpreter/BackendPasses.h

```diff
@@ -10,7 +10,8 @@
 #ifndef CLING_BACKENDPASSES_H
 #define CLING_BACKENDPASSES_H
 
-#include "llvm/IR/LegacyPassManager.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Passes/PassPlugin.h"
```

Comment:
Do we need anything from these headers? Instead, I believe we need the following:
```c++
#include "llvm/Analysis/CGSCCPassManager.h"
#include "llvm/Analysis/LoopAnalysisManager.h"
#include "llvm/Passes/PassManager.h"
```